From 6b2de4ec3816d4d8af30606593484f138efb9aff Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 1 Jun 2018 10:23:07 -0700
Subject: [PATCH] TRITON-457 sdc-scripts update in TRITON-380 broke cns and
 sdc-manatee Reviewed by: Marsell Kukuljevic <marsell@joyent.com> Approved by:
 Marsell Kukuljevic <marsell@joyent.com>

---
 deps/sdc-scripts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index 6bfa4a2..deefaef 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit 6bfa4a298d753685c6e083d10b70b5abdc39f66f
+Subproject commit deefaef587ed3bee2706cb6e53ee3468e682932e
-- 
2.21.0

